src/PrestaShopBundle/Twig/Component/Header.php

```diff
@@ -71,4 +64,77 @@ public function getJsDef(): array
     {
         return Media::getJsDef();
     }
+
+    public function getPsVersion(): string
+    {
+        return $this->psVersion;
+    }
+
+    public function getIsoUser(): string
+    {
+        return $this->context->getLanguage()->getIsoCode();
+    }
+
+    public function getCountryIsoCode(): string
+    {
+        return $this->context->getContext()->country->iso_code;
+    }
+
+    public function getLangIsRtl(): bool
+    {
+        return (bool) $this->context->getLanguage()->isRTL();
+    }
+
+    public function getShopName(): string
+    {
+        return $this->configuration->get('PS_SHOP_NAME');
+    }
+
+    public function getControllerName(): string
+    {
+        return htmlentities(Tools::getValue('controller'));
+    }
+
+    public function getImgDir(): string
+    {
+        return _PS_IMG_;
```

Comment:
I think you can improve this, when we look at how this const is built https://github.com/PrestaShop/PrestaShop/blob/3b478f53abbfc81e5c68ea128b6682730b78edde/config/defines_uri.inc.php#L46 you can see it actually concatenates `/img` to base URI which is defined here:
https://github.com/PrestaShop/PrestaShop/blob/72e70559775b5a3f2a652428016fcda794c2a742/config/config.inc.php#L124

So what you need to do here is get the current shop base URI and append `/img` to it this way the component won't depend on this old const anymore

The shop base URI is a variable we're gonna encounter in many templates and that should be easily reusable, but since the Shop context refacto is not done yet I guess you need to fetch it from the context for now and we'll try and improve it later